assembl/static2/js/app/pages/synthesis.jsx

```diff
@@ -103,15 +136,23 @@ export class DumbSynthesis extends React.Component<void, SynthesisProps, Synthes
               </Section>
             )}
             <Row className="background-grey synthesis-tree">
-              <Col md={3} className="affix">
-                {!sideMenuIsHidden && (
-                  <SideMenu
-                    rootIdeas={roots}
-                    descendants={descendants}
-                    synthesisPostId={synthesisPostId}
-                    ideaOnScroll={ideaOnScroll}
-                  />
-                )}
+              <Col
+                md={3}
+                className="affix"
+                style={{
+                  top: `${sideMenuTopPercentage} %`,
```

Comment:
Does it work with a space before `%` ? I never put space before unit in css.

Reply:
Yes it does